content/master/guides/composition-functions.md

```diff
@@ -0,0 +1,1002 @@
+Step 3/10 : FROM build AS build-venv
+ ---> 9b34960c88d7
+Step 4/10 : COPY requirements.txt /requirements.txt
+ ---> Using cache
+ ---> fae19dad52af
+Step 5/10 : RUN /venv/bin/pip install --disable-pip-version-check -r /requirements.txt
+ ---> Using cache
+ ---> f4b811c75812
+Step 6/10 : FROM gcr.io/distroless/python3-debian11
+ ---> 2a0e74a2b005
+Step 7/10 : COPY --from=build-venv /venv /venv
+ ---> Using cache
+ ---> cf727d3f20d3
+Step 8/10 : COPY . /app
+ ---> a044aef45e32
+Step 9/10 : WORKDIR /app
+ ---> Running in d08a6144815b
+Removing intermediate container d08a6144815b
+ ---> 7250f5aa653e
+Step 10/10 : ENTRYPOINT ["/venv/bin/python3", "main.py"]
+ ---> Running in 3f4d9dc55bad
+Removing intermediate container 3f4d9dc55bad
+ ---> bfd2f920c591
+Successfully built bfd2f920c591
+
+# Tag the Function.
+$ docker tag bfd2f920c591 example-org/xfn-quotable-simple:v0.1.0
+
+# Push the Function.
+$ docker push xpkg.upbound.io/example-org/xfn-quotable-simple:v0.1.0
+The push refers to repository [xpkg.upbound.io/example-org/xfn-quotable-simple]
+cf6d94b88843: Pushed
+77646fd315d2: Mounted from example-org/xfn-quotable
+50630ee42b6e: Mounted from example-org/xfn-quotable
+7e2cf97ed8c4: Mounted from example-org/xfn-quotable
+96e320b34b54: Mounted from example-org/xfn-quotable
+fba4381f2bb7: Mounted from example-org/xfn-quotable
+v0.1.0: digest: sha256:d8a6404e5fe38936aa8dadd861fea35ede0aded6168d501052f91cdabab0135e size: 1584
+```
+
+You can now use this Function in your Composition. The below example will create
+an `RDSInstance` using P&T Composition, then run the Function to annotate it
+with a quote.
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: example
+spec:
+  compositeTypeRef:
+    apiVersion: database.example.org/v1alpha1
+    kind: XPostgreSQLInstance
+  resources:
+    - name: rds-instance
+      base:
+        apiVersion: rds.aws.upbound.io/v1beta1
+        kind: Instance
+        spec:
+          forProvider:
+            dbName: exmaple
+            instanceClass: db.t3.micro
+            region: us-west-2
+            skipFinalSnapshot: true
+            username: exampleuser
+            engine: postgres
+            engineVersion: "12"
+      patches:
+        - fromFieldPath: spec.parameters.storageGB
+          toFieldPath: spec.forProvider.allocatedStorage
+      connectionDetails:
+        - type: FromFieldPath
+          name: username
+          fromFieldPath: spec.forProvider.username
+        - type: FromConnectionSecretKey
+          name: password
+          fromConnectionSecretKey: attribute.password
+  functions:
+  - name: quotable
+    type: Container
+    container:
+      image: xpkg.upbound.io/example-org/xfn-quotable-simple:v0.1.0
+      network:
+        policy: Runner
+```
+
+### Tips for New Functions
+
+Here are some things to keep in mind when building a Composition Function:
+
+* Your Function may be running as part of a pipeline. This means it's very
```

Comment:
```suggestion
* Your Function may be running as part of a pipeline. This means it's
```
unnecessary adverb. https://learn.microsoft.com/en-us/style-guide/word-choice/use-simple-words-concise-sentences

Reply:
This would be a good one to add to the style guide.